semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/structure/Type.scala

```diff
@@ -100,6 +100,13 @@ class Type(val wrapped: NodeSteps[nodes.Type]) extends AnyVal {
     new NodeSteps(
       raw
         .in(EdgeTypes.EVAL_TYPE)
-        .hasLabel(NodeTypes.IDENTIFIER, NodeTypes.CALL, NodeTypes.LITERAL)
+        .filterOnEnd(_.isInstanceOf[nodes.Expression])
         .cast[nodes.Expression])
+
+  def parameterOfType: NodeSteps[nodes.MethodParameterIn] =
```

Comment:
Concerning the naming convention: we do not have any other Steps named `of$nodeType`. Is it possible to just make it `parameter` or does that clash?

Reply:
I was using the same schema as for `expressionOfType`. But that also seems to be a deviation from the schema. I will change that.

Reply:
Ok. Approving now so you can merge when changed.